src/Strategy/Node.hs

```diff
@@ -216,6 +220,22 @@ data YarnVersion
   = V1
   | V2Compatible
 
+data NpmLockVersion
+  = NpmLockV1Compatible
+  | NpmLockV3Compatible
```

Comment:
- [ ] TODO: Change it to `NpmLockV3` instead, as we don't run npmLock analysis for v2 lockfile